persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_23/Ballerina.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+org = "foo"
+name = "persist_generate_23"
+version = "0.1.0"
+distribution = "2201.3.0"
+
+[build-options]
+observabilityIncluded = false
+
+[[platform.java11.dependency]]
+groupId = "mysql"
+artifactId = "mysql-connector-java"
+version = "8.0.29"
```

Comment:
I guess this is for the negative scenario testing

Reply:
updated in [d1e9e41](https://github.com/ballerina-platform/persist-tools/pull/52/commits/d1e9e4195fff39c00f1fce312aa40487f7196262)